src/base/Headers.hpp

```diff
@@ -161,14 +163,20 @@ inline int GetErrno() {
   if (retval >= 10000) {
     // Do some translation
     switch (retval) {
-    case WSAEWOULDBLOCK: return EWOULDBLOCK;
-    case WSAEADDRINUSE: return EADDRINUSE;
-    case WSAEINPROGRESS: return EINPROGRESS;
-    case WSAENOTSOCK: return ENOTSOCK;
-    case WSAECONNRESET: return ECONNRESET;
-    case WSAECONNABORTED: return ECONNABORTED;
-    default:
-      STFATAL << "Unmapped WSA error: " << retval;
+      case WSAEWOULDBLOCK:
```

Comment:
Just curious, did clang google format do this?

Reply:
Yep